src/solvers/dg/indicators.jl

```diff
@@ -33,7 +33,12 @@ function IndicatorHennemannGassner(equations::AbstractEquations, basis;
                                    alpha_max=0.5,
                                    alpha_min=0.001,
                                    alpha_smooth=true,
-                                   variable=first)
+                                   variable=nothing)
+  # check arguments
+  if isnothing(variable)
+    throw(ArgumentError("You must provide the keyword argument `variable`"))
+  end
+
```

Comment:
Why not just provide no default?
```suggestion
                                   variable)
```

Reply:
Thanks! Fixed in b5266b7.